documentation/docs/operation/commands/history.md

```diff
@@ -81,4 +81,29 @@ Using the same example, get the starting quote value. Convert the starting ETH a
 ```
 Return % = -0.562 USDT / 2539.13 USDT
 Return % = -0.0222 %
+```
+
+
+## Net Delta
+
+Net Delta is calculated based on the difference of starting balance and current balance, while Trade Delta is tokens acquired after executing trades. Both Net Delta or Trade Delta can be positive or negative (profit or loss).
+
+$Net Delta = Starting - Current$
+
+```
+Inventory:                                                                 
+      Market Asset  Starting   Current Net Delta Trade Delta               
+  0  binance   ETH   10.0000   10.0000    0.0000      2.0000               
+  1  binance  USDT  500.0000  499.9840   -0.0160   -407.7920               
+                                                                           
+```
+
+Using the same example, after executing these trades we can calculate Net Delta for base and quote assets, so Net Delta for base asset is **0.0000** because we started with `10.0000 ETH` and current also `10.000 ETH`. Net Delta for quote asset is **-0.0160 USDT** because we started with `500.0000 USDT` and current is `499.9840 USDT`, it means we obtained lost of **-0.0160 USDT** on quote asset.
```

Comment:
"it means we ~~obtained~~ lost ~~of~~ 0.0160 USDT on quote asset."

Reply:
Noted, thanks.